AGENTS.md

```
@@ -29,6 +29,16 @@ Environment variables are generally set in .env, not in Wrangler, except for cer
 | Build (local) | `npm run build-local`                                          |
 | Test          | `npm run test`                                                 |
 | Dev server    | `npx wrangler dev --local` (serves on `http://localhost:8787`) |
+<<<<<<< starlight-docs
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)